lib/env-options.js

```diff
@@ -0,0 +1,7 @@
+/* @flow */
+
+module.exports = {
+  encoding: process.env.dotenv_config_encoding,
```

Comment:
Let’s use uppercase to match our existing documentation as well as Node’s official docs. Also, a simple unit test will help with integration tests.